homeassistant/components/electric_kiwi/select.py

```diff
@@ -0,0 +1,69 @@
+"""Support for Electric Kiwi hour of free power."""
+from __future__ import annotations
+
+import logging
+
+from homeassistant.components.select import SelectEntity, SelectEntityDescription
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import EntityCategory
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import ATTRIBUTION, DOMAIN
+from .coordinator import ElectricKiwiHOPDataCoordinator
+
+_LOGGER = logging.getLogger(__name__)
+ATTR_EK_HOP_SELECT = "hop_select"
+
+HOP_SELECT = SelectEntityDescription(
+    entity_category=EntityCategory.CONFIG,
+    key=ATTR_EK_HOP_SELECT,
+    translation_key="hopselector",
+)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Electric Kiwi select setup."""
+    hop_coordinator: ElectricKiwiHOPDataCoordinator = hass.data[DOMAIN][entry.entry_id]
+
+    _LOGGER.debug("Setting up HOP entity")
```

Comment:
This is already logged?

Reply:
I'll update the text which I ended up doing in my next PR